gymnasium/spaces/box.py

```diff
@@ -256,7 +257,7 @@ def __repr__(self) -> str:
         """
         return f"Box({self.low_repr}, {self.high_repr}, {self.shape}, {self.dtype})"
 
-    def __eq__(self, other) -> bool:
+    def __eq__(self, other: object) -> bool:
```

Comment:
This is probably technically correct (and redundant), but for consistency it should be `other: Any`

Reply:
Good point, changed